includes/languages/english/extra_definitions/bootstrap/lang.zca_bootstrap_common.php

```diff
@@ -100,4 +100,11 @@
     'TEXT_HEADER_ARIA_LABEL_NAVBAR' =>'Navigation Bar',
     'TEXT_HEADER_ARIA_LABEL_LOGO' => 'Site Logo',
 ];
+
+// -----
+// included for account_history_info.php change in i.5.8b
+if (zen_get_zcversion() <= '1.5.8a') {
```

Comment:
Nope, just add that to the array above; the zc158 language processing will use that as a template-override.